src/devices/Pca95x4/Pca95x4.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
NIT: I think it would be easier to do what we do in other projects and simply exclude the default items and explicitly include what we want in the project.

Reply:
I'll need to review this as it was most likely caused while using the device-binding template and this was generated code as default.